source/client/HandView.py

```diff
@@ -100,16 +99,17 @@ def nextEvent(self):
                         )
                     self.hand_info = self.refreshXY(self.hand_info)
                 if self.prepare_card_btn.isOver(pos):
-                    # todo for Helen -
-                    print("Tried to prepare card, but not yet implemented")
-                    # todo for Sheri - need to update display so that prepared cards are obvious...
+                    user_input_cards = self.controller.automaticallyPrepareCards(self.gatherSelected())
+                    # The user_input_cards are a list of card/key option pairs ex. [[(0, None), [1,4,5,6,7,8,9,10,11,12,13]], [(2, 'Hearts'), [1,4,5,6,7,8,9,10,11,12,13]]]
+                    # TODO: for Sheri - need to get user input on what key to prepare user_input_cards (wild cards) in.
+                    # To prepare them when ready call self.controller.prepareCard(card, key)
+                    #
+                    # TODO: for Sheri - need to update display so that prepared cards are obvious...
```

Comment:
Planning to outline prepared cards in blue, instead of green.   Will also change "Move" button so prepared are to the right of selected cards.